### PR TITLE
[14.0][FIX] rma_sale, fix performance on search sale order lines

### PR DESCRIPTION
**Pull request by ChrisOForgeFlow**
Change domain to computed field, to allow document can be reprocessed, without this change on reprocess domain lost and search on all sale order lines

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
I think it makes sense

File: rma_sale/views/rma_order_line_view.xml [hunk 1]
```diff
@@ -33,11 +33,17 @@
                 </div>
                 <group name="main_info" position="inside">
                     <field name="sale_id" invisible="1" />
+                    <field
+                    name="sale_line_domain_ids"
+                    invisible="1"
+                    widget="many2many_tags"
```
Comment (LoisRForgeFlow) [inline rma_sale/views/rma_order_line_view.xml:39]: widget in an invisible field?

**Review by LoisRForgeFlow — APPROVED** [review]
Looks correct to me!

File: rma_sale/views/rma_order_line_view.xml (hunk 1 above)
Comment (ChrisOForgeFlow) [reply rma_sale/views/rma_order_line_view.xml:39]: I'm not sure if in this version can works without widget, but in previous version I need to put widget and domain works well

**Comment by AaronHForgeFlow** [comment]
Tested. Works ok. Can you please forward port it?

**Pull request merged**